Makefile

```diff
@@ -19,6 +19,12 @@ XGOIMAGEIOSSIM = statusteam/xgo-ios-simulator:$(XGOVERSION)
 
 DOCKER_IMAGE_NAME ?= status-go
 
+ROOT_DIR = $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+DOCKER_TEST_WORKDIR = /go/src/github.com/status-im/status-go/
+DOCKER_TEST_IMAGE  = golang:1.9
+DOCKER_TEST_EXEC = docker run --privileged --rm -it -v "$(ROOT_DIR):$(DOCKER_TEST_WORKDIR)" \
```

Comment:
Do we really need `DOCKER_TEST_EXEC`? I guess it could be simply:
```
docker-test:
    docker run --privileged --rm -it -v "$(shell pwd):$(DOCKER_TEST_WORKDIR)" -w "$(DOCKER_TEST_WORKDIR)" $(DOCKER_TEST_IMAGE)
```